apps/frontend/src/app/5_pages/MarketMakingPage/components/AdjustAndDepositModal/hooks/useGetMinimumReturn.ts

```diff
@@ -0,0 +1,18 @@
+import { useMemo } from 'react';
+
+import { Decimal } from '@sovryn/utils';
+
+import { decimalic } from '../../../../../../utils/math';
+
+const SLIPPAGE_AMOUNT = 5;
+
+export const useMinReturn = (amount: Decimal) => {
```

Comment:
Why is this a custom hook? It doesn't fetch anything and it doesn't interact with any other custom hooks either. `useMemo` here is unnecessary and it could be a util function instead.

Reply:
Yeah, there was probably an interaction or some sort of data fetching but was subsequently deleted